docs/Documentation/UserGuide/4-Operation Manual/8-TsFile Usage.md

```diff
@@ -620,4 +620,16 @@ and then call:
 TSFileConfig config = TSFileDescriptor.getInstance().getConfig();
 ```
 
+## Bloom filter
+
+Bloom filter check whether a given time series might in the tsfile before load metadata. This can improve the performance of loading metadata and skip tsfile that doesn't contains specified time series.
+If you want to learn more about its mechanism, you can refer to: [wiki page of bloom filter](https://en.wikipedia.org/wiki/Bloom_filter).
+
+#### configuration 
+you can control the false positive rate of bloom filter by the following parameter in the config file config file `tsfile-format.properties` which located at `/server/src/assembly/resources/conf` directory
```

Comment:
"config file" is repeated
"located" -> "is located"

Reply:
I will fix it